samples/word/50-document/manage-settings.yaml

```
@@ -0,0 +1,133 @@
+order: 6
+id: word-document-manage-settings
+name: Manage settings
+description: 'This sample shows how to add, get, and delete settings in the file.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#create-setting").click(() => tryCatch(createSetting));
+        $("#get-settings").click(() => tryCatch(getSettings));
+        $("#delete-all-settings").click(() => tryCatch(deleteAllSettings));
+
+        async function createSetting() {
+          await Word.run(async (context) => {
+            let key = $("#key")
+              .val()
+              .toString();
+            let value = $("#value")
+              .val()
+              .toString();
+            const settings = context.document.settings;
+            let setting = settings.add(key, value);
+            setting.load();
+            await context.sync();
+
+            console.log("Setting added:");
+            console.log(setting);
+          });
+        }
+
+        async function getSettings() {
+          await Word.run(async (context) => {
+            let settings = context.document.settings;
+            settings.load("items");
+            await context.sync();
+
+            const allSettings = [];
+            for (let i = 0; i < settings.items.length; i++) {
+              let setting = settings.items[i];
+              setting.load();
+              allSettings.push(setting);
+            }
+
+            await context.sync();
+
+            if (allSettings.length <= 0) {
```

Nit since the length can't be less than 0 (and we shouldn't imply otherwise).
```suggestion
            if (allSettings.length == 0) {
```